src/css/main.css

```diff
@@ -1 +1,40 @@
+.main {
+  font-family: 'Helvetica Neue Regular', 'Helvetica Neue';
+  font-weight: 200;
+  max-width: 900px;
+  margin: auto;
+  color: #333333;
+}
 
```

Comment:
what do you think about adding something like that:

html {
    font: 62.5%/1.35 'Helvetica Neue Regular', 'Helvetica Neue';
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
}

*, *:before, *:after {
    -webkit-box-sizing: inherit;
    -moz-box-sizing: inherit;
    box-sizing: inherit;
}

(same as the personal website)